src/modules/gnd_pos_control/GroundRoverPositionControl.cpp

```diff
@@ -135,11 +124,7 @@ GroundRoverPositionControl::GroundRoverPositionControl() :
 	_parameter_handles.l1_damping = param_find("GND_L1_DAMPING");
 	_parameter_handles.l1_distance = param_find("GND_L1_DIST");
 
-
-	_parameter_handles.airspeed_min = param_find("FW_AIRSPD_MIN");
-	_parameter_handles.airspeed_trim = param_find("FW_AIRSPD_TRIM");
 	_parameter_handles.airspeed_max = param_find("FW_AIRSPD_MAX");
```

Comment:
Definitely, but the basic version of a pixhawk slapped on an RC car should still be able to work.
How could I setup optical flow to work on a car like this?